nixos/modules/services/backup/sanoid.nix

```diff
@@ -176,11 +161,6 @@ in {
     # Implementation
 
     config = mkIf cfg.enable {
-      services.sanoid.settings = mkMerge [
-        (mapAttrs' (d: v: nameValuePair ("template_" + d) v.settings) cfg.templates)
```

Comment:
Why was this removed?

Reply:
So, while fixing the bug I was looking to strip unecessary complexity like using ˋmkMergeˋ and then when submiting this PR I thought that having a module changing its own user-faced option is unusual and confusing, and so far I don't see the added value when one can just merge everything in the call to ˋgenerators.toINIˋ. Sounds right for you?

Reply:
Oh, I'm discovering this new ˋsettingsˋ scheme. So indeed it's unusual but it's not using it that will soon be confusing.